### PR TITLE
[tracker] Update to v1.3.0. JB#53910

### PR DESCRIPTION
**Pull request by abranson**
Needed for Rygel upgrade for Tracker3
Switched to upstream build with meson.
Remove old git subtree

### CODE REVIEW

**Comment by saukko** [comment]
Extra points from removing Group and cleaning Source to have just {name}-{version}.tar.xz or so. :)

File: rpm/gssdp.spec [hunk 1]
```diff
@@ -37,31 +38,27 @@ Requires: pkgconfig
 Files for development with gssdp.
 
 %prep
-%setup -q -n %{name}-%{version}/%{name}
+%setup -q -n %{name}-%{version}/upstream
```
Comment (okodron) [inline rpm/gssdp.spec]:
```suggestion
%autosetup -n %{name}-%{version}/upstream
```

File: rpm/gssdp.spec [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Name:          gssdp
-Version:       1.0.1
+Version:       1.2.3
 Release:       1%{?dist}
```
Comment (okodron) [inline rpm/gssdp.spec]: Just `1` is enough I think

File: rpm/gssdp.spec [hunk 3]
```diff
@@ -37,31 +38,27 @@ Requires: pkgconfig
...
 
-#Remove libtool archives.
-find %{buildroot} -name '*.la' -exec rm -f {} ';'
+%meson_install
 
 %post -p /sbin/ldconfig
 
 %postun -p /sbin/ldconfig
 
 %files
```
Comment (okodron) [inline rpm/gssdp.spec:47]: Can you add license as well?
<code>
%license COPYING
</code>

**Comment by okodron** [comment]
Not sure that all these `Requires` are really necessary for `devel` sub-package.

**Comment by mlehtima** [comment]
Change submodule to github mirror.

**Review by okodron — APPROVED** [review]

File: rpm/gssdp.spec [hunk 4]
```diff
@@ -1,23 +1,20 @@
 Name:          gssdp
-Version:       1.0.1
-Release:       1%{?dist}
+Version:       1.2.3
```
Comment (okodron) [inline rpm/gssdp.spec]: Is it possible to update to the latest `1.3.0` release?

Comment (abranson) [reply rpm/gssdp.spec]: Done. The old mirror hadn't picked up these new releases.

Comment (mlehtima) [reply rpm/gssdp.spec]: Update also the PR title to match the new version.

**Review by okodron — APPROVED** [review]

**Review by mlehtima — APPROVED** [review]

**Comment by pvuorela** [comment]
LGTM

**Pull request merged**